src/main/java/seedu/address/model/country/CountryManager.java

```diff
@@ -14,13 +16,13 @@
     //TODO: Should include checking for 3-letter Country Code?
     public static final String MESSAGE_CONSTRAINTS = "Country code must be a valid 2-letter ISO3166 country code";
```

Comment:
This should probably be moved over to `Country.java` to be more consistent with the other classes.

Reply:
Ok done